ci/verify-build.sh

```diff
@@ -73,7 +72,7 @@ test_target() {
 
     if [ "$os" = "linux" ]; then
         # Test with the equivalent of __USE_TIME_BITS64
-        RUST_LIBC_UNSTABLE_LINUX_TIME_BITS64=1 $cmd
+        RUST_LIBC_UNSTABLE_LINUX_TIME_BITS64=1 $cmd --target-dir "target/$target-time64"
```

Comment:
Instead of the changes to this file, could you add `println!("cargo:rerun-if-env-changed=RUST_LIBC_UNSTABLE_LINUX_TIME_BITS64")` to build.rs?

Reply:
Do you just mean the right place within `build.rs`? If so, I would put it immediately before/after the env is read at `let linux_time_bits64 = env::var("RUST_LIBC_UNSTABLE_LINUX_TIME_BITS64").is_ok();` so it's easy to see they go together.

LGTM otherwise, note that CI will fail until a rust-lang/rust fix gets into nightly.

Reply:
That was my question.  I moved things around a bit to collect the linux_time_bits64 stuff in one place as much as possible.